HarpGuru/components/useQuizCycle/index.ts

```diff
@@ -0,0 +1,28 @@
+import { useGlobal, useDispatch } from 'reactn'
+import { useEffect } from 'react'
+
+// TODO: I don't think I should be importing from a
+// hooks file. The enum probably needs to be defined
+// elsewhere.
```

Comment:
Remove this TODO and reference in https://github.com/js-jslog/harpnative/issues/45

Reply:
Done